.github/workflows/release.yml

```diff
@@ -62,6 +62,8 @@ jobs:
           labels: ${{ steps.meta.outputs.labels }}
           cache-from: type=registry,ref=${{ env.IMAGE_NAME }}:edge
           cache-to: type=inline
+          sbom: true
+          provenance: mode=max
```

Comment:
Ich habe hier die Befürchtung, dass damit das re-tagging des Docker Image mit dem `latest` Tags nicht mehr funktionieren wird.

![image](https://github.com/user-attachments/assets/416bf528-0527-402e-bf25-e43f52a983b2)

Wenn der Output an Metadaten aus `sbom` und `provenance` beim _pre-release_ und _release_ build nicht exakt gleich ist (was ziemlich wahrscheinlich der Fall sein wird, weil das Datum ändert) wird hier ein neues Image gebaut. Der Lokale Cache den wir hier brauchen, können wir möglicherweise so auch nicht mehr verwenden.
>Note that adding attestations to an image means you must push the image to a registry directly, as opposed to loading the image to the local image store of the runner. This is because the local image store doesn't support loading images with attestations.

Der Release Build hier ist mir schon lange unsympathisch weil wir sehr viel Code und Annahmen haben um lediglich ein `latest` Tag hinzuzufügen. Dieser eine wichtiee Aspekt ist weder vom Job-Titel noch aus dem Code herauszulesen ist 🙈

PP/REQ: Mein Vorschlag wäre hier: Neuer Job und dann auf der Docker CLI `docker login`, `pull`, `tag` und `push` machen. Der Pre-Release würde bestehen bleiben.